contracts/teller-network/Escrow.sol

```diff
@@ -436,16 +436,15 @@ contract Escrow is Pausable, MessageSigned, Fees, Arbitrable {
      * @notice Set arbitration result in favour of the buyer or seller and transfer funds accordingly
      * @param _escrowId Id of the escrow
      * @param _releaseFunds Release funds to buyer or cancel escrow
+     * @param _arbitrator Arbitrator address
      */
-    function setArbitrationResult(uint _escrowId, bool _releaseFunds) external {
+    function setArbitrationResult(uint _escrowId, bool _releaseFunds, address _arbitrator) external {
         assert(msg.sender == address(arbitration)); // Only arbitration contract can invoke this
```

Comment:
~~Why do we need to pass the `_arbitrator` if we check here that the sender is `address(arbitration)`? ~~
Nvm, jus saw, `abitration` is the address of the contract.

Reply:
Also, this PR is to make it so that anyone can be an arbiter as long as it's not the buyer or seller? Is that so?